dev-requirements.txt

```diff
@@ -3,6 +3,8 @@ nose-exclude==0.4.1
 mock==1.3.0
 coverage==3.7.1
 tox==2.1.1
+pathlib
+reFILE
```

Comment:
It is currently being used in `tox.ini` to stash the coverage data of each `nosetests` environment (simple `cp` could suffice instead), and then also in the `coveragereport` task to pull all of the stashed data back into the working directory and rename it so that coverage combined picks them all up (I have only done a quick search for a better approach to achieve this.).  Very open to suggestions on alternatives.